crates/header-translator/translation-config.toml

```diff
@@ -1542,3 +1542,9 @@ skipped = true
 # `os_log_t` not defined; skip for now
 [class.MXMetricManager.methods.makeLogHandleWithCategory]
 skipped = true
+
+# The MPMediaQueryAddition definition does not have API_UNAVAILABLE annotations
+# where as the MPMediaItem does. This results in problematic conditional compliation:
+# https://github.com/phracker/MacOSX-SDKs/blob/041600eda65c6a668f66cb7d56b7d1da3e8bcc93/MacOSX11.3.sdk/System/Library/Frameworks/MediaPlayer.framework/Versions/A/Headers/MPMediaQuery.h#L103-L118
+[class.MPMediaItem.categories.MPMediaQueryAdditions]
```

Comment:
I suspect this is not the only case (it's just the only one caught since we're compiling for macOS), perhaps it would make sense to change `Stmt::Methods::category` to `Option<(ItemIdentifier<Option<String>>, Availability)>`?

This is also an argument for adding `Unavailable/PlatformCfg` to `ItemIdentifier`, then the problem would neatly go away.